src/foam/dao/EasyDAO.js

```diff
@@ -1026,14 +1047,14 @@ model from which to test ServiceProvider ID (spid)`,
         return this;
       },
       javaCode: `
-        if ( getInnerDAO() == null && getMdao() != null ) {
-          ((foam.dao.MDAO) getMdao()).addIndex(props);
-        } else {
-          Logger logger = (Logger) getX().get("logger");
-          if ( logger == null ) {
-            logger = new foam.nanos.logger.StdoutLogger();
+        if ( getMdao() != null ) {
```

Comment:
Are we no longer want to log when the nspec tries to add index to wrapper easy DAO? eg. no log for when adding index in localUserDAO?

Reply:
The  cmd(MDAO.GET_MDAO_CMD) will traverse the delegates and return if it his another EasyDAO or a real MDAO, so we can attempt to add indexes on nspecs such as localUserDAO, and if we can't then it warns.

Reply:
but we don't setMdao() when there is an innerDAO, do we? so getMdao() here will be null, no?

Reply:
yeah, I can just call the cmd on the delegate, no concern for the mdao, good call.